README.md

```diff
@@ -61,6 +61,7 @@ The following table lists the software versions NGINX Kubernetes Gateway support
 | NGINX Kubernetes Gateway | Gateway API | Kubernetes | NGINX OSS |
 |--------------------------|-------------|------------|-----------|
 | Edge                     | 0.8.0       | 1.22+      | 1.25.2    |
+| 0.6.0                    | 0.8.0       | 1.22+      | 1.25.2    |
```

Comment:
```suggestion
| Edge                     | 0.8.0       | 1.23+      | 1.25.2    |
| 0.6.0                    | 0.8.0       | 1.23+      | 1.25.2    |
```
Our minimum supported version is actually 1.23 not 1.22, I discovered this while working on the automated test for the minimum version

Reply:
Oh really? Do you know why that is?

Reply:
It's the CEL stuff. I misinterpreted the 0.8.0 blog post, it actually states that version 1.22 and older are no longer supported. 

https://gateway-api.sigs.k8s.io/blog/2023/0829-mesh-support/#kubernetes-122-and-older

Reply:
Ah, you're right.

Reply:
I think we made a mistake, in the 0.8.0 release notes its stated that 1.22 and older k8s versions are no longer supported:

> Kubernetes 1.22 and older

Unfortunately Gateway API v0.8.0 is not supported on these Kubernetes versions.
Gateway API v0.8.0 CRDs include CEL validation and cannot be installed on these
versions of Kubernetes. Note that Gateway API only commits to providing support
for the [5 most recent versions of Kubernetes](https://gateway-api.sigs.k8s.io/concepts/versioning/#supported-versions),
and thus these versions are no longer supported by Gateway API.

https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.8.0

There's an immediate error when you try and install the Gateway APIs on 1.22